src/utilities/RNG.cpp

```diff
@@ -75,7 +75,7 @@ namespace IBAMR
 /* Period parameters */
 #define N 624
 #define M 397
-#define MATRIX_A 0x9908b0df /* constant vector a */
+#define MATRIX_A 0x9908b0df   /* constant vector a */
```

Comment:
Is clang-format aligning comments now?

Reply:
I'm not totally positive where that one came from - I guess so?

Reply:
I like it! I used to do this stuff by hand and was sad that clang-format washed it away.